packages/zeego/src/menu/create-ios-menu/index.ios.tsx

```diff
@@ -242,12 +248,14 @@ If you want to use a custom component as your <Content />, you can use the menui
           ).targetChildren?.[0]
 
           if (imageChild) {
-            const { iosIconName } = imageChild.props
+            const { iosIconName, iosIconConfiguration } = imageChild.props
             if (iosIconName) {
-              // @deprecated  remove icon config for newer versions
               icon = {
-                iconType: 'SYSTEM',
-                iconValue: iosIconName,
+                type: 'IMAGE_SYSTEM',
```

Comment:
I'm not exactly sure why I even added this logic to the image; it's just duplicated from the `icon` logic above.

Reply:
Yeah, I wasn't sure about that either so I opted not to touch it. However, I've removed this block of code now - seems to work just fine.